M1/L1/index.html

```diff
@@ -1,56 +1,116 @@
+      Computer Science department has a course called
+      <a target="_blank" href="https://missing.csail.mit.edu/"
+        >"The Missing Semester of Your CS Education."</a
+      >
+      (we'll cite this often!)
+    </p>
 
-    <p>We built this mini-course to give a space to learn these fundamental tools in a deliberate, focussed way. We saw many people signing up for our Blockchain Frontend Bootcamp and having to learn these things while also trying to learn frontend developer and blockchain skills. This course hopes to pre-emptively help with many of the issues we've seen people struggle with over the years.</p>
+    <p>
+      This is not only makes starting to learn programming hard, it can also
+      lead to people to use fundamental tools poorly. If we are taught a task
+      only to accomplish another task, we'll typically only learn the minimum
+      effort that gets us to the second. This can often result in a tutorial
+      spiral where you end up completing X without digesting the fundamentals.
+      We are often taught the command line "good enough" to accomplish the job,
+      but not necessarily best practices. This often results in distaste and !
+      We can then end up compounding the problem by continuing to use these
+      essential tools in inefficient and sometimes agonizing ways.
+    </p>
 
-    <p>Even if you're very familiar with the technology in these lessons, we encourage you to flip through them. Along with introductory material, we also have tips and tricks for optimizing and improving your use of these tools. After all, if you're staring at a code editor for hours, shouldn't it be tricked out in the best way possible? Who knows, what you learn here could make your developer experience a lot more fun!</p>
+    <p>
+      We built this mini-course to give a space to learn these fundamental tools
+      in a deliberate, focused way. We saw many people signing up for our
+      Blockchain Frontend Bootcamp, having to learn the fundamentals while also
+      learning frontend development, along with blockchain skills. This course
+      hopes to pre-emptively help with many of the issues we've seen people
+      struggle with over the years.
+    </p>
 
-  <h2>A Few Things</h2>  
+    <p>
+      Even if you're very familiar with the technology in these lessons, we
+      encourage you to flip through them. Along with introductory material,
+      we’ve curated tips and tricks for optimizing and improving your use of
+      these tools. After all, if you're staring at a code editor for hours,
+      shouldn't it be tricked out in the best way possible? Who knows, what you
+      learn here could make your developer experience not only more efficient,
+      but fun!
+    </p>
 
-  <p>Luckily, to get comfortable you don't need to know that many things. It's just that they don't typically get explained together all at once.</p>
+    <h2>A Few Things</h2>
```

Comment:
I'm cool with a new header here, but perhaps more descriptive?

Reply:
I think <h2>A Few Things</h2> was pulled from the original looks to exists in 

basic-training/M1/L1/index.html main branch